deploy/helm/kuberhealthy/templates/configmap.yaml

```diff
@@ -30,4 +30,18 @@ data:
           description: Kuberhealthy shows that the cluster is not healthy
 {{- end -}}
 {{- end -}}
-
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: special-config
+data:
+  kuberhealthy.yaml: |-
+    listenAddress: ":8080"
+    enableForceMaster: true
+    logLevel: "info"
+    influxUsername: ""
+    influxPassword: ""
+    influxURL: ""
+    influxDB: "http://localhost:8086"
+    enableInflux: false
```

Comment:
Add some comments after each line here describing what it does.  You can use `#` to add a comment in this yaml

Reply:
Changes made with 6bf8b54